spyder/app/mainwindow.py

```diff
@@ -2776,13 +2776,25 @@ def _check_updates_ready(self):
             check_updates = box.is_checked()
         else:
             if update_available:
+                anaconda_msg = ''
+                if 'Anaconda' in sys.version or 'conda-forge' in sys.version:
+                    anaconda_msg = _("<b>Important note: </b> It seems that "
+                                     "you are using Spyder with Anaconda/"
+                                     "Miniconda, please <b>don't</b> use pip "
+                                     "to update as it would probably break "
+                                     "your installation.<br><br> Instead, "
+                                     "please wait until new conda packages "
+                                     "are made available and use <code>conda "

```

Comment:
Please change the text from this point to the end to

> are made available and use <b>conda</b> to perform the update.

The thing is people could have used conda-forge to install Spyder :-)